lib/stdlib/doc/src/erl_features.xml

```diff
@@ -63,7 +63,20 @@
       <name name="all" arity="0" since="OTP 25.0"/>
       <fsummary>Return all known features.</fsummary>
       <desc>
-        <p>Return a list of all known features.</p>
+        <p>Return a list of all known features.  This list will
+        include features that have been removed (status
+        <c>rejected</c>) and features that are no longer configurable
+        (status <c>permanent</c>).</p>
+      </desc>
+    </func>
+
+    <func>
+      <name name="configurable" arity="0" since="OTP 25.0"/>
+      <fsummary>Return all configurable features.</fsummary>
+      <desc>
+        <p>Return a list of all configurable features, i.e., features
```

Comment:
```suggestion
        <p>Return a list of all configurable features, that is, features
```

(Our documentation guidelines says that we should not use abbreviations such a "i.e." or "e.g.".)